src/mount/readdata_cache.h

```diff
@@ -65,13 +67,14 @@ class ReadCache {
 
 		Entry(Offset offset, Size requested_size)
 		    : offset(offset), buffer(), timer(), requested_size(requested_size),
-		      set_member_hook(), lru_member_hook() {}
+		      set_member_hook(), lru_member_hook(), mutexPtr(new std::mutex()) {}
```

Comment:
Well, I tried it, but it seems not easy to do without using the deleted function ‘std::mutex::mutex(const std::mutex&)' -> compile error. I think it can be done, but in more than one line or not looking so good.

Thanks anyway for the suggestion :)